android/src/main/kotlin/com/example/android_locator_plugin/constants.kt

```diff
@@ -0,0 +1,5 @@
+object Constants {
+    const val WHITE_SPACE = " "
+    const val BASE = 10
+    const val DECIMALS_NUMBER = 2
+}
```

Comment:
missed end of file extra line, check the other comment

Reply:
done